File: android/src/main/cn/io/github/nullptrx/pangleflutter/bean/PangleResult.java
package io.github.nullptrx.pangleflutter.bean;

import java.util.HashMap;
import java.util.List;

public class PangleResult {
  public int code;
  public String message;
  public Boolean verify;
  public Integer count;
  public List data;

  public HashMap<String, Object> toMap() {
    HashMap<String, Object> map = new HashMap<>();
    map.put("code", code);
    map.put("message", message);
    map.put("verify", verify);
    map.put("count", count);
    map.put("data", data);
    return map;
  }
}

File: android/src/main/cn/io/github/nullptrx/pangleflutter/bean/PangleResult.kt
package io.github.nullptrx.pangleflutter.bean

import java.util.HashMap

class PangleResult {

  var code = 0
  var message: String? = null
  var verify: Boolean? = null
  var count: Int? = null
  var data: List<*>? = null

  fun toMap(): HashMap<String, Any?> {
    val map = HashMap<String, Any?>()
    map["code"] = code
    map["message"] = message
    map["verify"] = verify
    map["count"] = count
    map["data"] = data
    return map
  }
}
